src/plugins/syscalls/syscalls.cpp

```diff
@@ -149,6 +180,15 @@ static GSList* create_trap_config(drakvuf_t drakvuf, syscalls *s, symbols_t *sym
 
             PRINT_DEBUG("[SYSCALLS] Adding trap to %s\n", symbol->name);
 
+            syscall_wrapper_t *wrapper = (syscall_wrapper_t *)g_malloc(sizeof(syscall_wrapper_t));
+
+            for (j=0; j<NUM_SYSCALLS; j++) {
+              if(strcmp(symbol->name,syscall_struct[j].name)==0) {
+                wrapper->sc=s;
+                wrapper->syscall_index=j;
+              }
+            }            
```

Comment:
What happens if the syscall wasn't found in search? Shouldn't continue at that point.